src/scenes/ProgressReports/Detail/WorkflowCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { Card, CardActions, StepperProps, Typography } from '@mui/material';
+import { ProgressReportStatus as Status } from '~/api/schema/schema.graphql';
+import { RelativeDateTime } from '~/components/Formatters';
+import { ButtonLink } from '~/components/Routing';
+import { ProgressReportDetailFragment } from './ProgressReportDetail.graphql';
+import { StatusStepper } from './StatusStepper';
+
+type WorkflowCardProps = {
+  report?: ProgressReportDetailFragment;
+} & StepperProps;
+
+export const WorkflowCard = ({ report, ...rest }: WorkflowCardProps) => {
+  const currentStatus = report?.status.value as Status | undefined;
+
+  const currentWorkflowEvent = report?.workflowEvents.at(
```

Comment:
Events are never current, they happen in an instant. Just like you wouldn't say "current update" but rather "last/latest update"

```suggestion
  const lastWorkflowEvent = report?.workflowEvents.at(
```

Reply:
Sorry about this, I have no Idea why this didn't get fixed the first time.  I could have sworn I fixed this and committed it.